### PR TITLE
Blackjack update

### PR DESCRIPTION
**Pull request by gazivodag**
isKnockedOut is no longer being used to swap menu entrys. So it was removed

Added other message to trigger knockout tick. This pickpockets the bandit when he aggros you preventing trading damage and better xp/h

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/blackjack/BlackjackPlugin.java [hunk 1]
```diff
@@ -94,9 +93,10 @@ public void onChatMessage(ChatMessage event)
 	{
 		if (event.getType() == ChatMessageType.SPAM)
 		{
-			if (event.getMessage().equals("You smack the bandit over the head and render them unconscious."))
+			final String msg1 = "You smack the bandit over the head and render them unconscious.";
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/blackjack/BlackjackPlugin.java]: this should be static final in global variables.

File: runelite-client/src/main/java/net/runelite/client/plugins/blackjack/BlackjackPlugin.java [hunk 2]
```diff
@@ -94,9 +93,10 @@ public void onChatMessage(ChatMessage event)
...
+			final String msg2 = "Your blow only glances off the bandit's head.";
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/blackjack/BlackjackPlugin.java]: same here

File: runelite-client/src/main/java/net/runelite/client/plugins/blackjack/BlackjackPlugin.java [hunk 3]
```diff
@@ -79,7 +78,7 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 
 		String option = Text.removeTags(event.getOption().toLowerCase());
 		String target = Text.removeTags(event.getTarget().toLowerCase());
-		if (isKnockedOut && nextKnockOutTick >= client.getTickCount())
+		if (nextKnockOutTick >= client.getTickCount())
 		{
 			MenuUtil.swap(client, "pickpocket", option, target);
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/blackjack/BlackjackPlugin.java]: We won't be using menu util to swap in the future, this was temporary.

**Comment by gazivodag** [comment]
Closing this PR and making it an option to toggle this.

Some people may prefer the more xp p/h at the cost of more trips

File: runelite-client/src/main/java/net/runelite/client/plugins/blackjack/BlackjackPlugin.java (hunk 3 above)
Comment (gazivodag) [reply runelite-client/src/main/java/net/runelite/client/plugins/blackjack/BlackjackPlugin.java]: Yeah. MenuManager isn't a fan of blackjacking unfortunately

**Pull request merged**